test/onnx/test_onnx_opset.py

```diff
@@ -378,6 +378,24 @@ def forward(self, input):
         x = torch.randn(2, 3, 4)
         check_onnx_opsets_operator(MyModule(), x, ops, opset_versions=[9, 10])
 
+    def test_fake_quantize_per_tensor(self):
```

Comment:
ONNX checker is already applied on ONNX models at export, which is covered in the onnxruntime test you've added.
Any ONNX models that do not pass checker cannot be successfully exported and executed by onnxruntime.

If you still want to keep this test, you should be able to create a list of ONNX ops (within the symbolic subgraph) and test it with 'check_onnx_opsets_opeator' through 'ops' argument. Note that this is only validating the ONNX subgraph ops created within the symbolic.

Reply:
If the onnxruntime is sufficient, I can remove this test.

Reply:
Removed. And added a test in test_models.py